static/js/script.js

```diff
@@ -1,44 +1,62 @@
-var queryLink	= document.getElementById('queryBuilderLink');
-var width = document.getElementById('width');
-var height = document.getElementById('height');
-var dateFrom = document.getElementById('dateFrom');
-var timeFrom = document.getElementById('timeFrom');
-var dateTo = document.getElementById('dateTo');
-var timeTo = document.getElementById('timeTo');
-var orientation = document.getElementById('orientation');
-var fit = document.getElementById('fit');
-var sort = document.getElementById('sort');
+var queryLink;
+var elements = [];
+var inputIds = [
+  'width',
+  'height',
+  'dateFrom',
+  'timeFrom',
+  'dateTo',
+  'timeTo',
+  'orientation',
+  'fit',
+  'order',
+  'sort'
+];
+
+document.addEventListener("DOMContentLoaded", function() {
```

Comment:
can be shortened to `document.addEventListener("DOMContentLoaded", init);`

Reply:
ah, of course - amended this